packages/medusa/src/api/routes/store/products/index.ts

```diff
@@ -24,3 +27,19 @@ export const defaultRelations = [
   "collection",
   "type",
 ]
+
+export * from "./list-products"
+export * from "./search"
+
+export type StoreProductsRes = {
+  product: Product
+}
+
+export class StorePostSearchRes {
+  hits: any[];
+  [k: string]: any
+}
+
+export type StoreGetProductsResponse = PaginatedResponse & {
```

Comment:
Should be:
```suggestion
export type StoreProductsListRes = PaginatedResponse & {
```
Make sure to fix in client as well :)